.github/actions/bench/action.yml

```diff
@@ -84,11 +87,20 @@ runs:
       run: |
         check-namespace
     - name: Store benchmark result
-      if: ${{ inputs.store_results == 'true' }}
       uses: benchmark-action/github-action-benchmark@v1
```

Comment:
can you pin this to a specific version please.

Reply:
I opened a separate PR to ping benchamrk and nix cache action now